sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextAnalyticsActions.java

```diff
@@ -55,8 +58,13 @@ public TextAnalyticsActions setDisplayName(String displayName) {
      * @param recognizeEntitiesActions the list of {@link RecognizeEntitiesAction} to be executed.
      *
      * @return the {@link TextAnalyticsActions} object itself.
+     *
+     * @throws IllegalArgumentException if more than one {@link RecognizeEntitiesAction} action are passed in.
+     * Currently service v3.1 only accepts up to one action per type and this check will be removed once the service
```

Comment:
no need to make promises here so please remove `and this check will be removed once the service officially removes this restriction.`